ocaml/driver/main_args.ml

```diff
@@ -1076,6 +1076,78 @@ let mk_no_flambda2_debug_concrete_types_only_on_canonicals f =
       Flambda2.Debug.Default.concrete_types_only_on_canonicals)
 ;;
 
+let mk_flambda2_inline_max_depth f =
+  "-flambda2-inline-max-depth", Arg.String f,
+  Printf.sprintf "<int>|<round>=<int>[,...]\n\
+      \     Maximum depth of search for inlining opportunities inside\n\
+      \     inlined functions (default %d) (Flambda 2 only)"
+    Clflags.Flambda2.Inlining.Default.max_depth
+;;
+
+let mk_flambda2_inline_cost arg descr ~default f =
+  Printf.sprintf "-flambda2-inline-%s-cost" arg,
+  Arg.String f,
+  Printf.sprintf "<float>|<round>=<float>[,...]\n\
+      \     The cost of not removing %s during inlining\n\
+      \     (default %f, higher = more costly) (Flambda 2 only)"
```

Comment:
```suggestion
      \     (default %.02f, higher = more costly) (Flambda 2 only)"
```
(Not certain both use the same unit, though.)

Reply:
Actually this needs 3 decimal places given the current figures.